### ISSUE TITLE
MacOS 10.10.2 Fail at haste-boot. Failed to download haste-libs-0.5.0.tar.bz2

### ISSUE THREAD

**Issue by DarwinSenior** [valderman/haste-compiler#302]
I installed haskell-platform, I followed the instruction using Cabal
And I got the following:
```bash
$ cabal update
$ cabal install haste-compiler
$ haste-boot
```
The following are the error message:
```
Downloading base libs from GitHub
Sending:
GET /haste-libs/haste-libs-0.5.0.tar.bz2 HTTP/1.1
Host: valderman.github.io


Creating new connection to valderman.github.io
Received:
HTTP/1.1 404 Not Found 
Server: GitHub.com
Content-Type: text/html; charset=utf-8
ETag: "550a6965-247d"
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; img-src data:; connect-src: 'self'
Content-Length: 9341
Accept-Ranges: bytes
Date: Mon, 23 Mar 2015 18:52:07 GMT
Via: 1.1 varnish
Age: 97
Connection: keep-alive
X-Served-By: cache-dfw1830-DFW
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1427136727.961983,VS0,VE0
Vary: Accept-Encoding


Failed to download http://valderman.github.io/haste-libs/haste-libs-0.5.0.tar.bz2: Not Found 
```

**Comment by jtojnar** [comment]
Did you, by any chance, have had development version from GitHub previously installed? Try [cleaning](https://github.com/valderman/haste-compiler/issues/281#issuecomment-68417394) the installation first

**Comment by DarwinSenior** [comment]
Sorry for the late reply, I still have same problem. It seems like `haste-libs-0.5.0.tar.bz2 not fount` is persistent.

**Comment by valderman** [comment]
What does `hastec --version` (or `hastec --version --unbooted` if that complains about not being properly booted) and `which haste-boot` show? No version on Hackage should request `haste-libs-0.5.0.tar.bz2`, so somehow the dev version of Hastec must be lying around, getting in the way.

**Comment by DarwinSenior** [comment]
So, the version is 0.5.0 and 
the `which haste-boot` is
```
/Users/DarwinSenior/Library/Haskell/bin/hastec
```
It all looks normal.

**Comment by worldsayshi** [comment]
So it's the github version then?

If so, try `haste-boot --force --local` from the source directory (I can't try it myself because I'm having other problems but its the way to do it according to the [readme](https://github.com/valderman/haste-compiler/blob/master/README.md#installation)).

**Comment by worldsayshi** [comment]
I think I just came out on the other end of this problem. Some things to keep in mind that I more or less contributed to solving it for me:
1. [here is a clean slate procedure if you've messed up your environment bad](https://github.com/valderman/haste-compiler/issues/281#issuecomment-68417394)
2. sandboxes can interfere in unexpected ways causing weird conflicts, I made sure to clear out my sandboxes before running the clean slate solution. Sandboxes in plural because I used ["advanced" sandbox features](https://www.haskell.org/cabal/users-guide/installing-packages.html#sandboxes-advanced-usage)
3. If using the github version it's:
```bash
$ cabal install
$ haste-boot --force --local
```
and not
```bash
$ cabal install haste-compiler
$ haste-boot
```
as described in the [readme](https://github.com/valderman/haste-compiler/blob/master/README.md#installation).
